services/web/src/utils/uploads.js

```diff
@@ -1,7 +1,7 @@
-import config from 'config';
+import { API_URL } from 'utils/env';
 
 export function urlForUpload(upload, thumbnail = false) {
-  let url = `${config.API_URL}1/uploads/${upload.hash}/image`;
+  let url = `${API_URL}1/uploads/${upload.hash}/image`;
```

Comment:
Let's make sure this path is correct. Both API_URL in env.conf and the Kubernetes deployment need to have that trailing slash

Reply:
They should always though no? A lot of other stuff would break if they didn't.

Reply:
we should just implement a "require("path").join" function and apply that to everything where we do any url building so this would never be an issue

Reply:
Yeah, I've run into issues with uploading not working after deploying. Let's get it in and I'll make sure to test.

Reply:
@dominiek I just updated to use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) which does path normalization for free so this shouldn't be an issue anymore